rate-exchange/rate-exchange.py

```
@@ -4,158 +4,174 @@
+
         # 最初と最後のレートを取得
         rates = []
         for log in yesterday_logs:
             try:
                 # "Current: 146.3400" のような形式から数値を抽出
-                if 'Current:' in log:
-                    rate_str = log.split('Current:')[1].split(',')[0].strip()
+                if "Current:" in log:
+                    rate_str = log.split("Current:")[1].split(",")[0].strip()
                     rates.append(float(rate_str))
             except:
```

Ruff (0.15.12)

[error] Do not use bare `except`

(E722)